src/renderer/hooks/useMapServerMutation.js

```diff
@@ -0,0 +1,26 @@
+// @ts-check
+import ky from 'ky/umd'
+import { useMutation, useQueryClient } from '@tanstack/react-query'
+
+// @ts-ignore
+const MAP_SERVER_URL = 'http://127.0.0.1:' + window.mapServerPort
+
+export function useMapServerMutation (mutationType, resourcePath) {
```

Comment:
similar question to https://github.com/digidem/mapeo-desktop/pull/708/files#r1050173134 about creating multiple hooks based on resource